.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 20.8b1 # Replace by any tag/version: https://github.com/psf/black/tags
+    rev: 22.3.0 #20.8b1 # Replace by any tag/version: https://github.com/psf/black/tags
```

Comment:
Can you update the other references to black to match this build number, in the conda files and in the GitHub workflow? (See fba532e for last time I changed the versions.)